src/megaclient.cpp

```diff
@@ -8731,5 +8859,89 @@ void MegaClient::cleanrubbishbin()
     reqs.add(new CommandCleanRubbishBin(this));
 }
 
+#ifdef ENABLE_CHAT
+void MegaClient::createChat(bool group, userpriv_vector *userpriv)
```

Comment:
Related to the previous comment. I can't see who is the owner of `userpriv`. On most functions, the SDK doesn't take the ownership of input parameters. However, this vector seems to be stored in `CommandChatCreate` and, depending on the result, it's propagated to `chatcreate_result` inside at `TextChat` object. I can't see a deletion on this vector nor in `megacli` nor in the intermediate layer.